homeassistant/components/switchbot/sensor.py

```diff
@@ -82,6 +84,18 @@
         state_class=SensorStateClass.MEASUREMENT,
         device_class=SensorDeviceClass.POWER,
     ),
+    "current": SensorEntityDescription(
+        key="current",
+        native_unit_of_measurement=UnitOfElectricCurrent.MILLIAMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.CURRENT,
+    ),
+    "voltage": SensorEntityDescription(
+        key="voltage",
+        native_unit_of_measurement=UnitOfElectricPotential.MILLIVOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.VOLTAGE,
+    ),
```

Comment:
In what unit of measurement is this the most usable? Because if this always measures 100V or 230V, we should not have the shown unit of measurement be millivolts. Rather we should set the `suggested_unit_of_measurement` to `VOLT`, so it will automatically come up like that

Reply:
Thank you for your valuable advice, I have fixed this problem. I also created a PR to bump the PySwitchbot at https://github.com/home-assistant/core/pull/131783. Could you please review the updated code at your earliest convenience?